src/wrappers/lambda/LambdaEventUtils.ts

```diff
@@ -21,6 +22,17 @@ class LambdaEventUtils {
 
     private constructor() {
     }
+    /**
+     * Checks if a propagated trace link exists in the incoming event
+     * @param originalEvent the original AWS Lambda invocation event
+     */
+    static extractTraceLinkFromEvent(originalEvent: any) {
+        try {
+            if (THUNDRA_TRACE_LINK_KEY in originalEvent) {
+                InvocationTraceSupport.addIncomingTraceLink(originalEvent[THUNDRA_TRACE_LINK_KEY].trace_link);
```

Comment:
It might be better to check whether `trace_link` is really exist in the `_thundra` key